OpenRA.Mods.Common/Traits/Player/PlayerStatistics.cs

```diff
@@ -133,6 +140,12 @@ public void WorldLoaded(World w, WorldRenderer wr)
 			UpdateEarnedThisMinute();
 			UpdateArmyThisMinute();
 		}
+
+		void INotifyResourceAccepted.OnResourceAccepted(Actor self, Actor refinery, int amount)
```

Comment:
Ah, and I forgot to mention: Using this interface is wrong, we'll need a new `INotifyCashGiven` interface.